jenkins/development.RHEL8.Dockerfile

```diff
@@ -0,0 +1,214 @@
+# RHEL 8 pbench development image
```

Comment:
In terms of the Jenkins environment, we are migrating away from the Fedora 33 container, but in the direction of an F34 or F35 one.  (We cannot run anything newer than F33 until the Jenkins executors are upgraded beyond RHEL 7.)

In terms of running the Server, the expectation was that it would be deployed on RHEL 8, so I started with a RHEL 8 platform instead of a Fedora one.

So, yeah, for the immediate future, this container will coexist with the Fedora one(s).  Really, we are pondering a matrix scenario, with several RHEL platforms and several Fedora platforms and testing "Pbench" on all of them.  But, if we like the containerized Server, then we'll probably stick with a single instance of that, so the matrix testing will involve only the Agent.

But, all that aside, I needed a base container to build the Server container on, and RHEL 8 seemed to be the Thing, so I put together this Dockerfile.